skyplane/cli/cli.py

```diff
@@ -218,47 +219,69 @@ def cp(
         )
         confirm_transfer(topo=topo, job=job, ask_to_confirm_transfer=not confirm)
 
-        transfer_stats = launch_replication_job(
-            topo=topo,
-            job=job,
-            debug=debug,
-            reuse_gateways=reuse_gateways,
-            use_bbr=cloud_config.get_flag("bbr"),
-            use_compression=cloud_config.get_flag("compress") if src_region != dst_region else False,
-            use_e2ee=cloud_config.get_flag("encrypt_e2e") if src_region != dst_region else False,
-            use_socket_tls=cloud_config.get_flag("encrypt_socket_tls") if src_region != dst_region else False,
-            aws_instance_class=cloud_config.get_flag("aws_instance_class"),
-            aws_use_spot_instances=cloud_config.get_flag("aws_use_spot_instances"),
-            azure_instance_class=cloud_config.get_flag("azure_instance_class"),
-            azure_use_spot_instances=cloud_config.get_flag("azure_use_spot_instances"),
-            gcp_instance_class=cloud_config.get_flag("gcp_instance_class"),
-            gcp_use_premium_network=cloud_config.get_flag("gcp_use_premium_network"),
-            gcp_use_spot_instances=cloud_config.get_flag("gcp_use_spot_instances"),
-            multipart_enabled=multipart,
-            multipart_min_threshold_mb=cloud_config.get_flag("multipart_min_threshold_mb"),
-            multipart_chunk_size_mb=cloud_config.get_flag("multipart_chunk_size_mb"),
-            multipart_max_chunks=cloud_config.get_flag("multipart_max_chunks"),
-            error_reporting_args=args,
-        )
+        # If size of transfer smaller than `use_overlay_limit_gb`, default to native cloud solution
+        if (
+            (job.transfer_size / GB) < cloud_config.get_flag("use_overlay_limit_gb")
```

Comment:
Rename this config flag to be: `use_overlay_limit_gb` to  `min_skyplane_threshold`